LearningMissionLab/SocialNetwork.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace LearningMissionLab
+{
+    public class SocialNetwork
```

Comment:
The SocialNetwork class should describe one particular social network. It should have the network name (consider adding an enum for that) and the link. In resume then you will have a list of the SocialNetwork objects

Reply:
Done.